### PR TITLE
Update runtime to 6.8 and add permission for inhibiting sleep.

### PR DESCRIPTION
**Pull request by Justinzobel**

### CODE REVIEW

File: org.kde.elisa.json [hunk 1]
```diff
@@ -29,6 +30,27 @@
...
+                "--enable-gpl",
+                "--enable-libvpx",
+                "--enable-libmp3lame",
+                "--enable-libvorbis",
+                "--enable-libopus"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://ffmpeg.org/releases/ffmpeg-4.4.5.tar.xz",
```
Comment (travier) [inline org.kde.elisa.json:49]: That's really old and we should likely use the extension instead.

**Comment by travier** [comment]
This one likely needs phonon back with the 6.8 runtime update like gwenview.

**Comment by tsdgeos** [comment]
> This one likely needs phonon back with the 6.8 runtime update like gwenview.

We did this on purpose so apps would fail to compile and thus we would be reminded to give them a backend, because when we had phonon on the runtime they compiled but failed at runtime since there was no backend and no single app would be adding a phonon backend because why would they?

File: org.kde.elisa.json (hunk 1 above)
Comment (Erick555) [reply org.kde.elisa.json:49]: I see it's explained in the commit, also: https://github.com/flathub/org.kde.okular/pull/387#issuecomment-2532806441

Comment (Justinzobel) [reply org.kde.elisa.json:49]: It's old for a reason, the new 6.8 KDE runtime uses the freedesktop 24.08 runtime which has ffmpeg 7. VLC isn't compatible with ffmpeg 7 yet.

The reason I chose 4.4.5 is I looked around and found Solus, Arch and Alpine all keep ffmpeg4 around for compatibility with VLC specifically.

Comment (travier) [reply org.kde.elisa.json:49]: Ouch, thanks, makes sense,

**Comment by travier** [comment]
Example that adds phonon and the backend: https://github.com/flathub/org.kde.gwenview/pull/166

**Comment by travier** [comment]
Ah look like Elisa uses qt-multimedia instead of phonon so this should be good.

**Comment by tsdgeos** [comment]
It uses vlc (wihtout phonon) or qtmultimedia.

"libvlc allows to play music in Elisa (otherwise it will use QtMultimedia)"

says the CMakeLists.txt 

If someone has time, ask the author if both are as good and we can save building ffmpeg and libvlc

**Pull request merged**